params/config.go

```diff
@@ -304,9 +304,25 @@ func NewNodeConfigWithDefaults(dataDir, fleet string, networkID uint64) (*NodeCo
 	nodeConfig.WhisperConfig.Enabled = true
 	nodeConfig.WhisperConfig.EnableNTPSync = true
 
+	nodeConfig.updatePeerLimits()
```

Comment:
hm, i thought you will add them to json file. or this one is a temporary solution?

Reply:
This is only for `NewNodeConfigWithDefaults`. status-react won't use this constructor.